dashboard/d_graphs.py

```diff
@@ -217,7 +220,7 @@ def generate_html(df) -> str:
         <table>
             <thead>
                 <tr>
-                    <th style='background-color: white;'>Topic</th>
+                    <th style='background-color: white; color: black'>Topic</th>
```

Comment:
Thanks for adding in this font colour change, this makes it a lot easier to see in dark mode!